crates/starknet-artifacts/tests/data/basic_package/src/lib.cairo

```diff
@@ -0,0 +1,20 @@
+mod hello_starknet;
+mod erc20;
+
+fn fib(a: felt252, b: felt252, n: felt252) -> felt252 {
+    match n {
+        0 => a,
+        _ => fib(b, a + b, n - 1),
+    }
+}
+
+#[test]
+fn test_fib() {
+    assert(fib(0, 1, 10) == 55, fib(0, 1, 10));
+}
+
+#[test]
+#[ignore]
+fn ignored_test() {
+    assert(1 == 1, 'passing');
+}
```

Comment:
Won't work with 2.3.0 (will be merged soon), you need to wrap it in `#[cfg(test)] mod tests { ... }`